aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -1055,10 +1066,26 @@ mod loader {
                 .no_credentials()
                 .load()
                 .await;
-            assert!(config.identity_cache().is_none());
             assert!(config.credentials_provider().is_none());
         }
 
+        #[cfg(feature = "rustls")]
```

Comment:
Does this cargo feature need to be set (which I assume it does since we also set it at line 1080 consistently)?

Reply:
Yes it fails without it and error indicates to enable it.